### ISSUE TITLE
Support parsing GS1 Web URIs

### ISSUE THREAD

**Issue by jodal** [jodal/biip#83]
GS1 has developed a standard URI format for encoding GS1 Application Identifiers named "GS1 Web URI".

The standard can be found here: https://www.gs1.org/standards/Digital-Link/1-0

If support for this in Biip would be useful for you, please reach out to me.

**Comment by dizhu1991** [comment]
Absolutely. Also support for other GS1 AIs would be fantastic, such as GLN, GRAI etc.

**Comment by jodal** [comment]
Just for the record, as an answer to the above comment, but otherwise unrelated to Web URI support:

Biip already supports all GS1 AIs, with specific support for many of them, quoting from the feature list in the project's README:

> - Parse AI 00 as SSCC.
> - Parse AI 01 and 02 as GTIN.
> - Parse AI 410-417 as GLN.
> - Parse dates/times into datetime.date/datetime.datetime values.
>   - Interpret the year to be within -49/+50 years from today.
>   - Interpret dates with day "00" as the last day of the month.
> - Parse variable measurement fields into Decimal values.
> - Parse discount percentage into Decimal values.
> - Parse amounts into Decimal values.
>   - Additionally, if py-moneyed is installed, parse amounts with currency into Money values.

**Comment by jodal** [comment]
Support for GS1 Web URIs have now been implemented. It will be part of the 4.0 release.
